jobsdb/backup.go

```diff
@@ -29,7 +29,7 @@ type backupSettings struct {
 }
 
 func (b *backupSettings) isBackupEnabled() bool {
-	return masterBackupEnabled && b.instanceBackupEnabled && config.GetString("JOBS_BACKUP_BUCKET", "") != ""
```

Comment:
don't we need a default (fallback) backup bucket anyway?

Reply:
True, but checking if `JOBS_BACKUP_BUCKET` env is set to determine if the backup is enabled does not seem right since we can take bucket details from the user as well right?

Reply:
Then jobsdb needs to consult another component for that?

Reply:
No, jobsdb only cares if master backup is enabled and instance backup is enabled and it's the job of fileuploader to provide bucket details to it right?

Reply:
> we can take bucket details from the user as well right?
But still, don't  we need to have a default bucket properly setup in a multitenant deployment, regardless of workspace-specific configurations?

Reply:
True, that assertion should happen independently of this right? Probably somewhere even before

Reply:
I agree with @atzoum. I think it is safer to keep the env check to ensure a default bucket is set for fallback, until we have a better way to do it.